test-bench-utils/lib/utils.test.js

```diff
@@ -0,0 +1,92 @@
+const utils = require('./utils');
+
+jest.mock('./frameworks', () => ({
```

Comment:
I think we should wire up these as tests in github actions, otherwise I worry they will get stale

Reply:
Agreed.